server/routes/notification.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const {
+  getNotifications,
+  createNotification,
+  getUnReadNotifications,
+  updateNotificationToRead,
+} = require("../controllers/notifications");
+const protect = require("../middleware/auth");
+
+router.route("/").get(protect, getNotifications);
+router.route("/unread/:id").get(protect, getUnReadNotifications);
+router.route("/:senderId").post(protect, createNotification);
```

Comment:
```suggestion
router.route("/").post(protect, createNotification);
```

Path paramets are used to identify a resource, consider sending the `senderId` in the `req.body` instead :)